docs/components/notification-badge.md

```diff
@@ -3,28 +3,35 @@ layout: layouts/component.njk
 title: Notification badge
 ---
 
-Use the notification badge component to notify users of something like an unread message.
+{% banner "HMRC Design Patterns has a similar component" %}
+[Notification badge](https://design.tax.service.gov.uk/hmrc-design-patterns/notification-badge/) in HMRC Design Patterns has a similar function and visual design to this component, and includes some guidance on when to use it.
 
-{% example "/examples/notification-badge", 125 %}
+This component is in the GOV.UK Design System [community backlog](https://design-system.service.gov.uk/community/backlog/) for review. 
+{% endbanner %}
 
-## When to use this component
+{% example "/examples/notification-badge", 125 %}
+### When to use
```

Comment:
Sorry, a fiddly thing but all these headings should be at second level (i.e. `###` rather than `##`) otherwise screenreaders can't parse the page properly